cmd/dosa/main_test.go

```diff
@@ -57,11 +57,10 @@ func TestHostOptionButNothingElse(t *testing.T) {
 func TestInvalidHost(t *testing.T) {
 	c := StartCapture()
 	exit = func(r int) {}
-	os.Args = []string{"dosa", "--host", "invalid-hostname.", "schema", "check", "--prefix", "foo", "../../testentity"}
+	os.Args = []string{"dosa", "--host", "", "schema", "check", "--prefix", "foo", "../../testentity"}
```

Comment:
The invalid hostname check is a different check than an empty hostname. We're more likely to see an invalid hostname than an empty one, and an invalid hostname should return an error that explains what is wrong very clearly.

Removing this check means we aren't returning a good error when the hostname is incorrect. Is this an upstream bug with yarpc, or are we interpreting the error from yarpc incorrectly now?